docs/configuration.rst

```diff
@@ -2289,3 +2289,34 @@ If true, need options like status, tags or links are collapsed and shown only af
 Default value: True
 
 Can be overwritten for each single need by setting :ref:`need_collapse`.
+
+
+.. _needs_per_page:
+
+needs_per_page
+~~~~~~~~~~~~~~
+
+.. versionadded:: 1.4.0
+
+Build list of ``json`` files that contain all found needs with the same ``docname``. The name of each file should match the ``docname``.
+
+This option works like needs_build_json. 
+
+Default: False
+
+
+.. _needs_per_page_build_path:
+
+needs_per_page_build_path 
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 1.4.0
+
+This option sets the location of list of ``json`` files that contain all found needs with the same ``docname``.
+
+Default value: need_per_page
```

Comment:
Please highlight path information like `need_per_page`.
Add also a `/` if it is a folder